cedar-policy-generators/src/schema.rs

```diff
@@ -152,6 +153,7 @@ fn arbitrary_typeofattribute_with_bounded_depth<N: From<ast::Name>>(
     Ok(json_schema::TypeOfAttribute {
         ty: arbitrary_schematype_with_bounded_depth::<N>(settings, entity_types, max_depth, u)?,
         required: u.arbitrary()?,
+        annotations: Annotations::new(),
```

Comment:
could generate some arbitrary annotations here for the schema roundtrip targets

Reply:
Yeah. I will create an issue and set up another PR for it. This PR is to ensure RFC 48 doesn't cause issues for existing properties.

Reply:
Issue: https://github.com/cedar-policy/cedar-spec/issues/493